See f0be25b6336db7492e47d2e8e72eb8af53b5506d in glibc

Upstream-Status: Inappropriate

locale/bits/types/locale_t.h | 1 +
1 file changed, 1 insertion(+)
create mode 100644 locale/bits/types/locale_t.h

diff -- git a/locale/bits/types/locale_t.h b/locale/bits/types/locale_t.h
new file mode 100644
index 0000000000. .b519a6c5f8
--- /dev/null
+++ b/locale/bits/types/locale_t.h
@@ -0,0+1 @@
+#include <xlocale.h>

--
2.14.3
